recipes/loompy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "loompy" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "36159c7d1423d4beb9848efc17ca5b2893398f1c7a6dccb408a413a112134c2b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - h5py
+    - typing
+
+test:
+  imports:
+    - loompy
+    - simplejson.tests
+
+about:
+  home: https://github.com/linnarsson-lab/loompy
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
The license file isn't being included in the source distribution since there is no `MANIFEST.in`. Please manually include the license file as described in the below link and raise an issue upstream, adding a comment in the recipe that includes the issue url:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually